docs/applications/nube-sdk/ui-slots.md

```
@@ -36,9 +36,26 @@ These are the slots that are available in checkout:
 This is their usual location:
 
+### Shipping Options slot
+
+The `after_shipping_description` slot allows you to add custom content after the shipping method description in the checkout page.
+
+```tsx
+import type { NubeSDK } from "@tiendanube/nube-sdk-types";
+import { Text } from "@tiendanube/nube-sdk-jsx";
+
+export function App(nube: NubeSDK) {
+	nube.render("after_shipping_description", [
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)